CHANGES.rst

```diff
@@ -21,6 +21,10 @@ splatalogue
 
 - Splatalogue table merging can now handle unmasked columns [#2136]
 
+jplhorizons
+^^^^^^^^^^^
+
+- Updated to use new JPL Horizons API. [#2154]
```

Comment:
I think here it's useful to be more specific (I don't see that the new API has a version, so I suggest to add the date instead)
```suggestion
- Updated to use new JPL Horizons API released on 2021-09-27. [#2154]
```

Reply:
Good point.  It does have a version (1.0), so I will add that.